src/pybind11/PyBindImath/PyBindImathBox.cpp

```diff
@@ -14,23 +14,35 @@ void register_box(pybind11::class_<T>& c)
     c.def(pybind11::init<>())
     .def(pybind11::init<const V &>())
     .def(pybind11::init<const V &, const V &>())
+    .def(pybind11::init<const T &>())  // Copy constructor
+    .def(pybind11::init<const std::pair<V, V> &>())  // Constructor from pair of points
     .def(pybind11::self == pybind11::self)
     .def(pybind11::self != pybind11::self)
     .def("makeEmpty", &T::makeEmpty, "makeEmpty() make the box empty")
     .def("makeInfinite", &T::makeInfinite, "makeInfinite() make the box cover all space")
-    .def_readwrite("min", &T::min)
-    .def_readwrite("max", &T::max)
-    .def("extendBy", pybind11::overload_cast<const T &>(&T::extendBy), "extendBy(point) extend the box by a box")
+    .def("min", &T::min, "min() returns the minimum point of the box")
+    .def("max", &T::max, "max() returns the maximum point of the box")
+    .def("setMin", &T::setMin, "setMin(point) sets the minimum point of the box")
```

Comment:
There is no `setMin()` or `setMax` method on the Matrix classes, you'll need a function, like:

        .def("setMin", [](T& self, const V& v) { self.min = v; },"setMin(point) sets the minimum point of the box")
```suggestion
    .def("setMin", [](T& self, const V& v) { self.min = v; },"setMin(point) sets the minimum point of the box")

```